src/database/migrations/20221012064037-create-review-request-views.js

```diff
@@ -0,0 +1,59 @@
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("review_request_views", {
+      review_request_id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.BIGINT,
+        references: {
+          model: "review_requests",
+          key: "id",
+        },
+        onUpdate: "CASCADE",
+        onDelete: "CASCADE",
+      },
+      site_id: {
```

Comment:
actually idt we need to store this cos the review request holds both site and user id

Reply:
update: the rr table holds siteId and _requestor_ id (so we might still need the user id)

Reply:
Yeah we will still need the user id as it can be someone that is completely not involved in the review request.